src/RedisSubscriber.js

```diff
@@ -48,10 +48,8 @@ export default class RedisSubscriber implements Subscriber {
     await promisify(cb => this.redis.subscribe(channel, cb))();
   }
 
-  subscribe(
-    channel: Channel,
-    parseMessage: ?(string) => any = this._parseMessage,
-  ) {
+  subscribe(channel: Channel, options: Object) {
```

Comment:
you can more strongly type `options` here if you want

Reply:
how?

Reply:
don't call my bluff :P aaaa

```
type Options = {
  parseMessage?: (msg: string) => any
}
``